sematic/metrics/func_success_rate_metric.py

```diff
@@ -0,0 +1,78 @@
+# Standard Library
+from datetime import datetime
+from typing import Optional, Tuple
+
+# Third-party
+from sqlalchemy.orm import Query, Session, joinedload
+
+# Sematic
+from sematic.abstract_future import FutureState
+from sematic.abstract_system_metric import AbstractSystemMetric
+from sematic.db.models.run import Run
+from sematic.metrics.metric_point import MetricType
+
+FINAL_STATES = {FutureState.RESOLVED, FutureState.FAILED, FutureState.NESTED_FAILED}
```

Comment:
Optionally: FINAL_STATES = [state.value for state in {FutureState.RESOLVED, FutureState.FAILED, FutureState.NESTED_FAILED}], so it saves multiple invocations on line 74.